runtime/queries/dbml/highlights.scm

```diff
@@ -0,0 +1,48 @@
+
+; comments highlighting
+(comment) @comment
+
+; keyword highlighting
+(keyword_def) @keyword
+(keyword_enum) @keyword
+(keyword_ref) @keyword
+
+; identify blocks and definitions
+(block) @class
+(definition) @function
+
+; for identifiers
+(identifier) @variable
+(type) @keyword
+
+; Highlight special types for database/data types
+("Project" ) @type
+("Table" ) @type
+("TableGroup" ) @type
+("database_type" ) @variable
+
+; Index related highlighting
+(index_block) @class
```

Comment:
`@class` isn't a capture name that we use, see https://docs.helix-editor.com/master/themes.html#syntax-highlighting

Reply:
I've noticed that the 'block' seems to have no noticeable effect on the syntax highlighting, removed it